contracts/core/native_fallback/L2/NativeProver.sol

```diff
@@ -55,26 +55,11 @@ contract NativeProver is Ownable, INativeProver {
         uint256 indexed _destinationChainID, uint256 indexed _blockNumber, bytes32 _L2WorldStateRoot
     );
 
-    /**
-     * @notice Block number is too recent to prove
-     */
-    error NeedLaterBlock(uint256 _inputBlockNumber, uint256 _nextProvableBlockNumber);
-
-    /**
-     * @notice Block number is older than currently proven block
-     */
-    error OutdatedBlock(uint256 _inputBlockNumber, uint256 _latestBlockNumber);
```

Comment:
Used in the async approach if we tried to prove against an older block than the latest cache one.